tests/unit/api/main.py

```diff
@@ -572,7 +578,7 @@ def test_node_connect_network_no_such_node(self):
         network_create_simple('hammernet', 'anvil-nextgen')
 
         with pytest.raises(api.NotFoundError):
-            api.node_connect_network('node-98', '99-eth0', 'hammernet') # changed
+            api.node_connect_network('node-98', '99-eth0', 'hammernet')  # changed # noqa
```

Comment:
Quoting Nicholas Matsuura (2016-08-02 14:18:48)

> Same issue as above I think. These lines are fine without # changed, I
> think we can probably get rid of the comment to shorten the line.

@gsilvis, what are your thought on this? IIRC you added those comments
as a band-aid on the fact that there's a lot of difficult-to-tease-apart
redundancy in these tests. IMO it doesn't help all that much, and I'd be
in favor of just removing the `# changed`.